src/index.js

```diff
@@ -0,0 +1,303 @@
+import './index.css';
+
+/**
+ *
+ * @param {String} selector
+ * @param {HTMLElement} parentTree
+ * @returns
+ */
+function $select(selector, tree = document.body) {
+  return tree.querySelector(selector);
+}
+
+/**
+*
+* @param {String} selector
+* @param {HTMLElement} parentTree
+* @returns
+*/
+function $selectAll(selector, tree = document.body) {
+  return tree.querySelectorAll(selector);
+}
+
```

Comment:
Good use of JQuery. Kindly try to use DOM in the global environment

Reply:
Hi @cyrilmuchemi 

this is not a JQuery property but rather a custom implementation to avoid repeating the same code over and over.

Reply:
Okay. i appreciate the clarification :smiley: